### DISCUSSION TITLE
ROR

### DISCUSSION THREAD

**Discussion by BreakersRC** [artisan-roaster-scope/artisan#1097] [General]
I am operating artisan via my Mac through a USB port and during my roast I am unable to see a projected ROR throughout my roasting. I have tried a few different configurations within the settings and am still unable to figure it out. As of right now im roasting completely based off color, time, and FC. I have a HOTTOP KN-8828B-2K+.

**Comment by roasterdave** [comment]
>I am unable to see a projected ROR throughout my roasting

Just to be sure, do you see BT projections?  Is it only the deltaBT (RoR) projections that you do not see?  As described under "QUADRATIC & ROR PROJECTS" in this [Artisan Blog post](https://artisan-roasterscope.blogspot.com/2022/03/artisan-v260.html), the RoR projection lines are not drawn before 5min into the roast.

If we need to go further then some basic information is always needed.  What version of Artisan are you using?  If you are unsure go to Help>> About.  What macOS are you running?  Thanks for identifying the roaster.  

It is also helpful to attach your current Artisan settings file (as exported via menu Help >> Save Settings) and an example Artisan *.alog profile.  Note that you must either add a `.txt` extension or zip the files before posting them here.  Do not attach the files in an email reply.  You should drag & drop, select or paste them onto your reply on this website.

**Reply by BreakersRC** [reply]
I am running Mac OS Ventura. I also downloaded the most recent artisan for Mac which I believe is artisan Mac 2.8.2.dmg. Currently the only way that I can see my ROR is through the analysis tool after I have already finished my roast. Thank you for the help

**Reply by roasterdave** [reply]
You need to attach your current Artisan settings file *.aset (as exported via menu Help >> Save Settings) and an example Artisan *.alog profile. Note that you must either add a .txt extension or zip the files before posting them here. Do not attach the files in an email reply. You should drag & drop, select or paste them onto your reply on this website.

**Reply by BreakersRC** [reply]
I am having an issue even trying to drop in file in which you could help me further. Does artisan have a call support? I can even find an up to date video or webpage describing how to resolve this issue.

**Reply by poundy** [reply]
I'm sorry but this is the support you can get for Artisan. It's an open source project, you should not expect there to be a helpdesk for it sitting waiting to take calls. 

You need to copy two files to some place on your Mac. You then need to rename them by putting .txt on the end. 
You then need to log back in here, and drag those two files into the next reply box. 

Tell us what part you get stuck with.

**Comment by BreakersRC** [comment]
[artisan-settings.txt](https://github.com/artisan-roaster-scope/artisan/files/10712737/artisan-settings.txt)

**Reply by poundy** [reply]
thins one is good ! The other one is not, it didn't appear to attach a file. We need a copy of a "not working" previous roast file.

**Reply by roasterdave** [reply]
@BreakersRC  The settings file came through okay. However, to help we still need you to attach an example Artisan *.alog profile file so we can see what is going on.

**Comment by BreakersRC** [comment]
artisan-settings.txt
